chart/templates/prometheus-servicemonitor.yaml

```diff
@@ -0,0 +1,42 @@
+{{/*
```

Comment:
I wonder if we could base the contents of this off of https://github.com/hashicorp/vault-secrets-operator/blob/ab5766f4ebbde57e9b2cbae518fe8f7985ece94d/config/prometheus/monitor.yaml#L1-L0 - that would align more closely with a kustomize based deployment